app/javascript/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Greeting from './Greeting';
+
+function App() {
+  return (
+    <main>
+      <div className="container">
+        <Greeting />
+      </div>
+    </main>
+  );
```

Comment:
- [ ] Great job with the project. As per project requirements, the `App` component is supposed to have a route that leads to the `Greeting` component. Kindly check and get that added.
![image](https://user-images.githubusercontent.com/62849872/213641568-a5bf3b2e-93ba-4292-acbb-2605a1a610c4.png)
`Create the Greeting component that will display a greeting. Set it up as a route in your App component.
`